### DISCUSSION TITLE
Feature for training left and right hands spacebar use

### DISCUSSION THREAD

**Discussion by Penguinflash** [monkeytypegame/monkeytype#1097] [Ideas]
Hello. I would like to increase my typing speed. I am Colemak-DH user. I'm actually using a custom configuration on GNU/Linux that is sort of a merger between Colemak and Latam-Colemak.

My problem is that I use the spacebar with only one hand. The correct thing to do is to use both hands (The opposite hand to the last character used).

I know that it is not possible to recognize by software which hand I am using, because the spacebar key is only one.

Will it be possible to offer an option to remap Spacebar in AltGr and have the software assign a hand to it?

It would look like this:

**Spacebar** - Left Hand spacebar
**AltGr** - Right Hand spacebar

If it's an awkward position, on my system I can reassign all the right hand keys a little more to the right and AltGr will be easier to press with right thumb.

Thank you for your attention to the suggestions.

PD: I have a conventional keyboard, I have also seen some keyboards (40%-60%) with the space bar "split" in two parts. I guess this suggestion will be useful also for those who have such a keyboard.

**Comment by unrealapex** [comment]
>My problem is that I use the spacebar with only one hand. The correct thing to do is to use both hands (The opposite hand to the last character used).

There is no real rule for this, do whatever you feel comfortable with.

**Comment by frogfrogfrogfrogfrog** [comment]
I figure that using both thumbs for the space bar seems quite useless. Opposite shift seems a bit more viable since it puts less strain on the pinkies but I think that the thumbs don't really need to switch since they never get strained (in my experience at least).
